packages/browserify/examples/01-simple-js/package.json

```diff
@@ -8,6 +8,7 @@
     "start": "node ./build.js && serve ."
   },
   "dependencies": {
-    "metamask-logo": "^2.2.1"
+    "lavamoat-browserify": "^16.0.0",
```

Comment:
updated in https://github.com/LavaMoat/LavaMoat/pull/476/commits/b5432db8576bd29415515118da6299dabb50f4a5 including docs

Reply:
> Did you experience problems running these examples? I test my code against example `01` constantly, never experiencing any problems..

just making sure @weizman you're adding `lavamoat-browserify` too right in this example 🤔
i checked `yarn global add lavamoat-browserify@15.5.0` (since 16 was wrongly introduced) but it has no binaries

Reply:
ok ^ was just wondering how we're testing the example without `lavamoat-browserify` as a dependency

e.g. possibly locally

<img width="788" alt="Screenshot 2023-03-28 at 11 37 37" src="https://user-images.githubusercontent.com/1881059/228210415-684d2157-cd8e-4ac3-8a35-6b148c16526f.png">

but yep it's not pos globally (since no binaries)

so deffo want it as a dependency (devDep as you pointed out)

so think PR now looks good to merge if you're happy